common.yml

```diff
@@ -641,6 +641,37 @@ functions:
               --package-file-suffix "" \
               release.msi
 
+  "sign linux packages":
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
+        script: |
+          echo "${signing_auth_token_server_4_4}" > ./signing_auth_token
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
```

Comment:
```suggestion
```
Let's not make this one silent, so that we can see if it prints out any messages!

Reply:
Should we keep the `silent: true` in the `sign msi installer` task?